RPG Wizardry/Assets/Scripts/Player/PlayerInput/SpellCraftingInput.cs

```diff
@@ -57,8 +57,12 @@ private void MouseInputs()
 
                 if (results.Count > 0)
                 {
-                    draggedImage = results[0].gameObject.GetComponent<Image>();
-                    mousePos = Input.mousePosition;
+                    if(results[0].gameObject.tag == "Bar")
```

Comment:
Use a [TagSelector]-annotation and a Serialized String (See ACollectable. Annotation can be put on either string or string[]) for tags, to ensure they use existing and valid tags